exampleVault/Advanced Examples/DnD 5e Ability Scores and Modifiers.md

```diff
@@ -0,0 +1,38 @@
+---
+STR: 8
+DEX: 16
+CON: 11
+INT: 12
+WIS: 14
+CHR: 11
+---
+
+This example calculates a DnD character's ability modifiers from the ability scores and displays it all in a list.
+
+The modifiers are stored in `memory` and not in the frontmatter. This reduces clutter but requires some hidden fields that calculate the values while the fields in the list just format the value to always contain a `+/-` sign.
+
+
+## Character ability scores
+
+```meta-bind
+VIEW[ floor(({STR}-10)/2) ][math(hidden):memory^STR_mod]
+```
+```meta-bind
+VIEW[ floor(({DEX}-10)/2) ][math(hidden):memory^DEX_mod]
+```
+```meta-bind
+VIEW[ floor(({CON}-10)/2) ][math(hidden):memory^CON_mod]
+```
+```meta-bind
+VIEW[ floor(({INT}-10)/2) ][math(hidden):memory^INT_mod]
+```
+```meta-bind
+VIEW[ floor(({CHR}-10)/2) ][math(hidden):memory^CHR_mod]
+```
+
+- STR: `VIEW[**{STR}**][text(renderMarkdown,class(score))]` (`VIEW[ concat(isPositive({memory^STR_mod})?'+':'', string({memory^STR_mod}))][math(class(moddifyer))]`)
```

Comment:
you add a CSS class called `moddifyer`. Shouldn't it be called `modifier` and you didn't include any CSS for it.

Reply:
ah sorry, both are correct. just leftover from by own document. I also style the elements, but that didn't seem super relevant to the example